packages/now-cli/src/util/dev/server.ts

```diff
@@ -85,6 +87,11 @@ import {
 } from './types';
 import { ProjectSettings } from '../../types';
 
+const frameworkList = _frameworks as Framework[];
```

Comment:
Why is the cast necessary? Does `@vercel/frameworks` not properly set the type?

Reply:
@TooTallNate Indeed, I have been saying that since its inception. Its source is written as JSON, not TS.

Reply:
Oh I think we can actually solve this by defining a default export in `.d.ts` 🤔

Reply:
Maybe not because we use named exports so we can use the CJS "default" export as far as I can see

https://github.com/vercel/vercel/blob/f1b426097ac1022fb42da0fb7e198287831dc197/packages/frameworks/index.d.ts

Reply:
Let's convert `@vercel/frameworks` to TS in another PR 👍